utility.py

```
@@ -49,6 +48,172 @@
+    # ── GitHub ────────────────────────────────────────────────────────────
+
+    def create_github_issue(self, title: str, body: str, labels: list[str] | None = None) -> None:
+        """
+        Create a GitHub issue on the TanjunBot/new_tanjun repository.
+
+        This is a **blocking** call (uses PyGithub).  Run via
+        ``run_blocking()`` when called from async code.
+        """
+        g = Github(self._github_token)
+        repo = g.get_repo("TanjunBot/new_tanjun")
+        issue_labels: list[github.Label.Label] = []
```

Flake8 (7.3.0)

[error] undefined name 'github'

(F821)